data/data_collection_test.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from data.data_collection import *
+
+
+class DataCollectionTest(unittest.TestCase):
+    """
+    Class that tests the functions in the data_collection.py.
+    """
+    def test_set_page(self):
+        """
+        Test the set_page() function on edge cases where the input page is less
+        than one.
+        """
+        data_collector = DataCollector('mock',
+                                       '2000-01-01T00:00:00Z',
+                                       '2010-01-01T00:00:00Z')
+        data_collector.set_page(1)
+        with self.assertRaises(ValueError):
+            data_collector.set_page(0)
+        with self.assertRaises(ValueError):
+            data_collector.set_page(-100)
+
+    def test_get_pull_request_review_time(self):
+        """
+        Test the logic of computing the pull request review time.
+        """
+        mock_pull_request_info = {
+            'created_at': '2018-09-16T00:20:58Z',
+            'closed_at': '2020-05-18T05:21:24Z'
+        }
+        pull_request_created_time = to_timestamp(
+            mock_pull_request_info['created_at'])
+        pull_request_closed_time = to_timestamp(
+            mock_pull_request_info['closed_at'])
+        pull_request_review_time = pull_request_closed_time - \
+            pull_request_created_time
+        self.assertEqual(pull_request_review_time, 1589779284-1537057258)
```

Comment:
What you are verifying here?
You should use mock_pull_request_info to mock the response of pull request and when you call the pull request, verify that not mock_pull_request_info.